static/styles/_components/_header.scss

```diff
@@ -143,3 +143,7 @@
     border-right: none;
   }
 }
+
+.section-header {
```

Comment:
Maybe move this to the `_typography.scss` partial and keep this one for the site header styles.